Jenkinsfile

```diff
@@ -15,6 +24,18 @@ node('docker && imgtec') {
                     [$class: 'PruneStaleBranch'],
                 ],
             ])
+            // Skip CI if changes in README, else continue building
+            sh 'git show --pretty="format:" --name-only --no-merges 89ffc511f185058ac6bf3fdaa44c932227d3416e > commandResult'
```

Comment:
All of this information should already be in the jenkins scm object as it is able to display a list of changes files on the web page.